src/components/ec-phone-number-input/types.ts

```diff
@@ -0,0 +1,77 @@
+import type { InputHTMLAttributes } from 'vue';
+
+import type { Maybe } from '../../../global';
+import type { InputFieldEvents } from '../ec-input-field/types';
+
+export interface PhoneNumberCountry {
+  areaCode: string,
+  text: string,
+  countryCode: string
+}
+
+export interface PhoneNumberCountryItem {
+  areaCode: PhoneNumberCountry['areaCode'],
+  iconPath: string,
+  id: PhoneNumberCountry['countryCode'],
+  name: PhoneNumberCountry['text'],
+  text: string,
```

Comment:
I just noticed that the `text` from original type is mapped to `name` and `text` is redeclared. Could you double check that?

Reply:
This was created intentionally to support search functionality in country dropdown.